db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create!(name: "hoge",
+             email: "hogehoge@hoge.com",
```

Comment:
存在しないドメインである`@example.com`を使った方がメールの誤送信の心配が無くなります。
```suggestion
             email: "hogehoge@example.com",
```

Reply:
ご指摘ありがとうございます。
修正いたしました。7740418